.circleci/config.yml

```
@@ -0,0 +1,97 @@
+---
+version: 2
+
+references:
+  images:
+    middleman: &MIDDLEMAN_IMAGE hashicorp/middleman-hashicorp:0.3.41
+
+  cache:
+    rubygem: &RUBYGEM_CACHE_KEY static-site-gems-v1-{{ checksum "content/Gemfile.lock" }}
+
+jobs:
+  deploy-website:
+    docker:
+      - image: *MIDDLEMAN_IMAGE
+    environment:
+      NO_WARM: true # we don't warm the cache to parallelize wgets in CircleCI
```

```suggestion
      NO_WARM: true # tells deploy.sh to skip the big recursive wget, so we can parallelize it.
```

I was confused by this earlier, and didn't realize it was a signal to our own script rather than a Circle thing.